index.js

```diff
@@ -85,7 +93,8 @@ Watcher.prototype.build = function Watcher_build(filePath) {
     .then(totalTime)
     .then(appendFilePath)
     .then(triggerChange, triggerError)
-    .then(verboseOutput.bind(this));
+    .then(verboseOutput.bind(this))
+    .then(cleanup);
```

Comment:
We don't have access to `run` in finally.  Seems weird to stash it in a closure.

Can you think of a better way?